src/graphics/program-lib/standard.js

```diff
@@ -283,9 +422,10 @@ pc.programlib.standard = {
         for (p in pc._matTex2D) {
             mname = p + "Map";
             if (options[mname + "VertexColor"]) {
-                cname = mname + "Channel";
+                cname = mname + "VertChannel";
```

Comment:
I think we should have:

diffuse
diffuseMap
diffuseMapChannel
diffuseVertexColor
diffuseVertexColorChannel